Set report output & skip comment

rcrowe opened the pull request:
Allow the report to be generated & accessed, but no PR comment raised; this allows more complex workflows to use this action for rendering only.

rcrowe committed:
Set rendered report as an output

rcrowe committed:
Skip creating a comment

milesj commented on index.ts:
```diff
@@ -89,6 +89,7 @@ async function run() {
 	try {
 		const accessToken = core.getInput('access-token');
 		const limit = Number(core.getInput('limit'));
+		const skipComment = core.getInput('skip-comment') === 'true';
```
You can use `getBooleanInput` I believe.

milesj commented on index.ts:
```diff
@@ -118,6 +119,13 @@ async function run() {
+
+		if (skipComment) {
+			core.debug('Skipping comment creation');
+			core.setOutput('comment-created', 'false');
```
Maybe call `saveSummary` here also?

rcrowe replied:
Makes sense 👍🏻

milesj commented:
@rcrowe Looks good, just a few Qs.

rcrowe commented:
Thanks for the comments, will get them sorted tomorrow 🧹

rcrowe committed:
Use string for default value

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

> A string representing the default value

rcrowe committed:
Save summary even if comment skipped

Address PR comments:
- Call saveSummary
- Use getBooleanInput

milesj commented:
awesome stuff
